readme.md

```diff
@@ -10,13 +11,13 @@
 ```bash
 $ npm install --global posthtml-cli
 ```
-> **Note:** This project is compatible with node v8+
```

Comment:
can this be returned?

Reply:
returned ? but it wont be compatible ! right ?

Reply:
> **Note:** This project is compatible with node v10+

Reply:
ohh okok 
i will add it